src/js/index.js

```diff
@@ -0,0 +1,35 @@
+import '../sass/style.scss';
+
+const data = [
+  { playerName: 'player1', score: 18 },
+  { playerName: 'player2', score: 1 },
+  { playerName: 'player3', score: 18 },
+  { playerName: 'player4', score: 199 },
+  { playerName: 'player5', score: 18 },
+];
+
+function display(data) {
```

Comment:
Let's use ES6 (arrow function) instead of using the `function` keyword.

![image](https://user-images.githubusercontent.com/31897434/128061071-a54dbec6-1b5c-44fc-8048-4a90a381fa1b.png)

Reply:
* Done 👍